testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/EEConcurrencyExecutorShutdownTestCase.java

```diff
@@ -0,0 +1,230 @@
+
+        final WildFlyManagedConfiguration config = createConfiguration("domain.xml", "host.xml", getClass().getSimpleName());
+        config.setAdminOnly(true);
+        config.setReadOnlyDomain(true);
+        config.setReadOnlyHost(true);
+
+        // Trigger the servers to fail on boot if there are runtime errors
+        String hostProps = config.getHostCommandLineProperties();
+        hostProps = hostProps == null ? "" : hostProps;
+        config.setHostCommandLineProperties(hostProps + "\n-Djboss.unsupported.fail-boot-on-runtime-failure=true");
+
+        this.addTestResources();
+    }
+
+    @After
+    public void removeTestResources() throws IOException, MgmtOperationException {
+        try {
+            // /profile=default/subsystem=ee/context-service=default:remove()
+            ModelNode removeContextService = createOpNode("profile=default/subsystem=ee/context-service=default", REMOVE);
+            executeForResult(removeContextService, masterClient);
+            // /profile=default/subsystem=ee/managed-executor-service=default:remove()
+            ModelNode removeManagedExecutorService = createOpNode("profile=default/subsystem=ee/managed-executor-service=default", REMOVE);
+            executeForResult(removeManagedExecutorService, masterClient);
+        } finally {
+            testSupport.stop();
+        }
+    }
+
+    @AfterClass
+    public static void tearDownDomain() throws Exception {
+        try {
+            testSupport = null;
+            domainMasterLifecycleUtil = null;
+            DomainTestSuite.stopSupport();
+        } finally {
+            cleanFile(tmpDir);
+        }
+    }
+
+    private void addTestResources() throws IOException, MgmtOperationException {
+        // /profile=default/subsystem=ee/context-service=default:add(jndi-name=java:jboss/ee/concurrency/context/default, use-transaction-setup-provider=true)
+        ModelNode addContextService = createOpNode("profile=default/subsystem=ee/context-service=default", ADD);
+        addContextService.get("jndi-name").set("java:jboss/ee/concurrency/context/default");
+        addContextService.get("use-transaction-setup-provider").set(true);
+        executeForResult(addContextService, masterClient);
+        // /profile=default/subsystem=ee/managed-executor-service=default:add(jndi-name=java:jboss/ee/concurrency/executor/default, context-service=default, hung-task-threshold=60000, keepalive-time=5000)
+        ModelNode addManagedExecutorService = createOpNode("profile=default/subsystem=ee/managed-executor-service=default", ADD);
+        addManagedExecutorService.get("jndi-name").set("java:jboss/ee/concurrency/executor/default");
+        addManagedExecutorService.get("context-service").set("default");
+        addManagedExecutorService.get("hung-task-threshold").set(60000);
+        addManagedExecutorService.get("keepalive-time").set(5000);
+        executeForResult(addManagedExecutorService, masterClient);
+    }
```

Comment:
@dcihak  I don't know if there is a specific reason for adding/removing the resources manually in the test instead of adding directly the ee subsystem configuration in the standard-domain.xml file. 
The performance penalty using the subsystem configured by default for other tests is minimal, but we could gain less code for the test, removing `addTestResources()` and `removeTestResources()` methods. It will also remove the need for executing management operations after checking that user case under test was correct. I don't have a strong opinion on it, it is just a suggestion.